tensorrt_llm/_torch/attention_backend/sparse/deepseek_v4/flash_mla.py

```
@@ -0,0 +1,934 @@
+        ]
+        max_lse = torch.stack(finite_pool_lses).amax(dim=0)
+        pool_weights = [torch.exp(pool_lse - max_lse) for pool_lse in finite_pool_lses]
+        denominator = torch.stack(pool_weights).sum(dim=0)
+        output = sum(
+            torch.where(
+                torch.isfinite(pool_lse).unsqueeze(-1),
+                pool_output,
+                torch.zeros_like(pool_output),
+            )
+            * pool_weight.unsqueeze(-1)
+            for pool_output, pool_lse, pool_weight in zip(pool_outputs, pool_lses, pool_weights)
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)